src/tactic/rcases.lean

```diff
@@ -931,5 +931,36 @@ add_tactic_doc
   decl_names := [`tactic.interactive.obtain],
   tags       := ["induction"] }
 
+/--
+The `rsuffices` tactic is an alternative version of `suffices`, that allows the usage
+of any syntax that would be valid in an `obtain` block. This tactic just calls `obtain`
+on the expression, and then `swap`.
```

Comment:
This docstring is not really accurate anymore after you changed the implementation.